assignment1/fc_net.py

```diff
@@ -99,6 +99,46 @@ def update_parameters(self, parameters, grads, learning_rate):
 
         return parameters
 
+    def L_layer_model(self, X, Y, layers_dims, learning_rate, num_iterations, batch_size):
+        """
+
+        :param X: the input data, a numpy array of shape (height*width , number_of_examples)
```

Comment:
In our implementation we assume the n_examples is the first dimension, no? If yes, we need to reshape

Reply:
we need to be consistent in all of our code you are right